cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -78,8 +81,8 @@
   private final File pluginArchive;
   private final Map<String, LocalizeResource> resourcesToLocalize;
 
-  private InputFormatProvider inputFormatProvider;
-
+  // key is input name, value is the MapperInput (configuration info) for that input
```

Comment:
It is still not clear what the input name is, or is it? Whatever Input.getName() returns?